src/i18n/es-es/index.js

```diff
@@ -191,6 +191,8 @@ export default {
         connect_wallet: 'Conectar billetera',
         view_address: 'Ver dirección',
         disconnect: 'Desconectar',
+        disable_wallet_extensions: 'Desactiva los monederos de extensión o establece Brave Wallet como predeterminado en la configuración del monedero del navegador para utilizar Brave Wallet.',
+        enable_wallet_extensions: 'Habilitar la extensión MetaMask y establecer por defecto `preferir extensiones` en la configuración del monedero del navegador para utilizar el monedero MetaMask.',
```

Comment:
If this text were part of an action triggered by a UI widget (like a button) it would be fine, but looking at the context in which it is used as a notification to the user, the text has an imperative tone, and that in Spanish changes a little bit:

```suggestion
        disable_wallet_extensions: 'Desactive los monederos de extensión o establezca Brave Wallet como predeterminado en la configuración del monedero del navegador para utilizar Brave Wallet.',
        enable_wallet_extensions: 'Habilite la extensión MetaMask y establezca por defecto `preferir extensiones` en la configuración del monedero del navegador para utilizar el monedero MetaMask.',
```